app/models/job_offer.rb

```diff
@@ -67,6 +67,10 @@ def register_new_visitor
     self.visit_counter = visit_counter + 1
   end
 
+  def self.find_active_offers_to_apply_by(a_log_user=nil)
```

Comment:
Creo que la frase sería "find active offers to be applied by". Y el parámetro debería ser sólo a_user, el modelo no tiene que saber si hay usuario logueados o no